package/utils/util-linux/Makefile

```diff
@@ -172,7 +172,7 @@ endef
 define Package/fdisk
 $(call Package/util-linux/Default)
   TITLE:=manipulate disk partition table
-  DEPENDS:= +libblkid +libsmartcols +libfdisk
+  DEPENDS:= +libblkid +libsmartcols +libfdisk +libncursesw
```

Comment:
https://github.com/lede-project/source/blob/master/package/libs/ncurses/Makefile#L42
It's an alias and I think it's worth preparing packages for unicode, for now it's just a cosmetic thing.